tests/scripts/all.sh

```diff
@@ -1246,6 +1246,14 @@ component_test_default_cmake_gcc_asan_new_bignum () {
     tests/context-info.sh
 }
 
+component_test_crypto_with_static_key_slots() {
+    msg "build: crypto config + MBEDTLS_PSA_STATIC_KEY_SLOTS"
+    scripts/config.py crypto
```

Comment:
At least
```suggestion
    scripts/config.py crypto_full
```

We need to validate compatibility with `MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS`, at least. Probably other things I'm not thinking of to start with.